core/ui-common/src/main/kotlin/com/wire/android/ui/common/chip/WireFilterChip.kt

```diff
@@ -0,0 +1,117 @@
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.ui.common.chip
+
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material3.FilterChip
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.SelectableChipColors
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.wire.android.ui.common.R
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.common.preview.MultipleThemePreviews
+import com.wire.android.ui.theme.WireTheme
+import com.wire.android.ui.theme.wireTypography
+
+@Composable
+fun WireFilterChip(
+    label: String,
+    isSelected: Boolean,
+    modifier: Modifier = Modifier,
+    isEnabled: Boolean = true,
+    onSelectChip: (String) -> Unit = {}
+) {
+
+    val rotationAngle by animateFloatAsState(
+        targetValue = if (isSelected) 0f else 45f,
+    )
+
+    FilterChip(
+        modifier = modifier
+            .height(25.dp)
+            .wrapContentWidth(),
+        onClick = { onSelectChip(label) },
+        label = {
+            Text(
+                text = label,
+                style = MaterialTheme.wireTypography.button03
+            )
+        },
+        enabled = isEnabled,
+        selected = isSelected,
+        colors = SelectableChipColors(
+            containerColor = Color.Transparent,
+            labelColor = colorsScheme().onBackground,
+            leadingIconColor = colorsScheme().onBackground,
+            trailingIconColor = colorsScheme().onBackground,
+            disabledContainerColor = Color.Transparent.copy(alpha = 0.5f),
+            disabledLabelColor = colorsScheme().onPrimaryVariant.copy(alpha = 0.5f),
+            disabledLeadingIconColor = colorsScheme().onPrimaryVariant.copy(alpha = 0.5f),
+            disabledTrailingIconColor = colorsScheme().onPrimaryVariant.copy(alpha = 0.5f),
+            disabledSelectedContainerColor = colorsScheme().primary.copy(alpha = 0.5f),
+            selectedContainerColor = colorsScheme().primaryVariant,
+            selectedLabelColor = colorsScheme().primary,
+            selectedLeadingIconColor = colorsScheme().primary,
+            selectedTrailingIconColor = colorsScheme().primary,
+        ),
+        trailingIcon = {
+            Icon(
+                modifier = Modifier
+                    .size(13.dp)
```

Comment:
```suggestion
                    .size(dimensions().spacing12x)
```
<img width="222" alt="image" src="https://github.com/user-attachments/assets/0625b5a3-651c-4634-bf2f-550c23e0c2f5" />